src/cloudant/_common_util.py

```diff
@@ -268,16 +268,24 @@ def get_docs(r_session, url, encoder=None, headers=None, **params):
 #pylint: disable=unused-argument
 def append_response_error_content(response, **kwargs):
     """
-    Provides a helper to add HTTP response error and reason messages.
+    Provides a helper to act as callback function for the response event hook
+    and add a HTTP response error with reason message to `response.reason`.
+    The `response` and `**kwargs` are necessary for this function to
```

Comment:
I guess in the long run it really doesn't matter because I think these functions aren't included with our docs.  So this is really a non-blocking comment.